fastlane/fastlane
# [frameit] Add force_resize_background option

### Checklist

### Motivation and Context

This pull request increases developer control on how background images are used for framed screenshots. 


### Description

This pull request preserves frameit's default behavior in which the background image is resized keeping the aspect ratio if the height of the background image does not equal the height of the screenshot.
This pull request adds a new configuration setting named `force_resize_background` to frameit.
If `force_resize_background` is set to true then frameit always resizes the background image ignoring aspect ratio (i.e. regardless of whether the height of the background equals the height of the screenshot).  


### Testing Steps


I ran the existing tests.
I ran frameit locally with the configuration set to true and false and not set, and observed that the resulting framed screenshots were as expected.
